onnxruntime/python/tools/quantization/notebooks/imagenet_v2/mobilenet.ipynb

```diff
@@ -322,9 +320,43 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "session_quant = onnxruntime.InferenceSession(\"mobilenet_v2.uint8.onnx\")\n",
+    "session_quant = onnxruntime.InferenceSession(\"mobilenet_v2_uint8.onnx\")\n",
     "run_sample(session_quant, 'cat.jpg', categories)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3 Convert the models to ort format."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This step is optional, we will convert the *mobilenet_v2_float.onnx* and *mobilenet_v2_uint8.onnx* to ORT format, to be used in mobile applications."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!{sys.executable} -m onnxruntime.tools.convert_onnx_models_to_ort --optimization_level basic ./"
+   ]
```

Comment:
Are we expecting they'll use a NNAPI? If not we should use 'all' rather than 'basic'.

Reply:
This model is optimized for NNAPI, so I would expect we can add NNAPI in the example

Reply:
People blindly copy examples and there's nothing here that alerts them that they need to think about the correct optimization level to use. 

Can we please add a comment explaining why we're using 'basic' and saying 'all' should be used in NNAPI won't be involved? Or a link to the doco on choosing the optimization level is fine too.

Reply:
Added some comments